spec/factories/users.rb

```diff
@@ -23,13 +23,13 @@
       trn_verified { true }
     end
 
-    trait :with_application do
+    factory :participant do
```

Comment:
I'm wondering about the naming, can we avoid participant factories possibly, as it's not a concept in NPQ, happy to chat about other options